core/tauri/src/app.rs

```diff
@@ -388,6 +388,14 @@ impl<R: Runtime> ManagerBase<R> for App<R> {
 macro_rules! shared_app_impl {
   ($app: ty) => {
     impl<R: Runtime> $app {
+      #[cfg(feature = "updater")]
```

Comment:
I think we need the __updater-docs flag here too so it's included in docs.rs, right?
On my phone rn so I can't really see the context/surrounding code.

Reply:
oops forgot about it again, thanks!